watermarker/src/commonMain/kotlin/watermarkers/text/PlainTextWatermarker.kt

```diff
@@ -1,29 +1,29 @@
 /*
- * Copyright (c) 2023-2025 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
+ * Copyright (c) 2025 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
```

Comment:
This PR removed the date range. Please undo this change:
```suggestion
 * Copyright (c) 2023-2025 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
```